src/modules/feature-modules/innovator/pages/innovation/record/section-edit.component.ts

```diff
@@ -26,204 +27,218 @@ export class InnovationSectionEditComponent extends CoreComponent implements OnI
   baseUrl: string;
 
   sectionsIdsList: string[];
-  wizard: WizardEngineModel;
-
+  sectionQuestionsIdList: string[];
+  wizard: WizardIRV3EngineModel;
+  sectionStatus: keyof typeof INNOVATION_SECTION_STATUS = 'UNKNOWN';
   saveButton = { isActive: true, label: 'Save and continue' };
   submitButton = { isActive: false, label: 'Confirm section answers' };
 
+  isChangeMode: boolean = false;
+
   sectionSubmittedText: string = '';
 
   displayChangeButtonList: number[] = [];
 
+  // Flags
+  isInnovatorType: boolean;
+
   constructor(private activatedRoute: ActivatedRoute) {
     super();
 
     this.innovation = this.stores.context.getInnovation();
     this.sectionId = this.activatedRoute.snapshot.params.sectionId;
     this.baseUrl = `/innovator/innovations/${this.innovation.id}/record/sections/${this.sectionId}`;
 
-    this.sectionsIdsList = getInnovationRecordConfig().flatMap(sectionsGroup =>
-      sectionsGroup.sections.map(section => section.id)
-    );
+    this.sectionsIdsList = this.stores.schema.getIrSchemaSubSectionsIdsListV3();
+    this.sectionQuestionsIdList = this.stores.schema.getIrSchemaSectionQuestionsIdsList(this.sectionId);
+
     this.wizard = this.stores.innovation.getInnovationRecordSectionWizard(this.sectionId);
+    this.wizard.currentStepId = this.activatedRoute.snapshot.params.questionId;
 
     this.isArchived = this.innovation.status === 'ARCHIVED';
 
+    // Flags
+    this.isInnovatorType = this.stores.authentication.isInnovatorType();
+
     this.setBackLink('Go back', this.onSubmitStep.bind(this, 'previous'));
   }
 
-  private getNextSectionId(): string | null {
+  private getNextSectionId(): string {
     const currentSectionIndex = this.sectionsIdsList.indexOf(this.sectionId);
-    return this.sectionsIdsList[currentSectionIndex + 1] || null;
+    return this.sectionsIdsList[currentSectionIndex + 1] ?? '';
   }
 
   ngOnInit(): void {
-    const sectionIdentification = this.stores.innovation.getInnovationRecordSectionIdentification(this.sectionId);
+    const sectionIdentification = this.stores.schema.getIrSchemaSectionIdentificationV3(this.sectionId);
 
     const savedOrSubmitted = !this.isArchived ? 'submitted' : 'saved';
 
     this.sectionSubmittedText = sectionIdentification
       ? `You have ${savedOrSubmitted} section ${sectionIdentification?.group.number}.${sectionIdentification?.section.number} '${sectionIdentification?.section.title}'`
       : '';
 
-    combineLatest([
```

Comment:
Why did you remove the combineLatest? Removing it the error of fetching data makes more sense on the second subscribe than the first since it the first ain't fetching data

Reply:
True, the error is in the wrong place indeed.
I just tried with both subscriptions on combineLatest again and it works properly, will adapt.